entity_gym/environment.py

```diff
@@ -90,6 +90,7 @@ class ObsFilter:
 
 
 @dataclass
+
 class CategoricalAction:
     actions: List[Tuple[EntityID, int]]
```

Comment:
The use of EntityID here seems strange, like we are mixing up the entities with the players which should be totally distinct?  Is there something else going on here I don't understand?  is it because in codecraft you have multiple characters controlled by a single player, and so they are all issued command simultaneously?

Reply:
There is no multi-agent support currently, observation/action is for a single player. You can sort of do multiple agents by by using a VecEnv where each player is exposed as its own environment, see the multi_snake env. I'm hoping this will be sufficient for now, but I would like to have more comprehensive multi-agent support a la PettingZoo at some point. So the EntityIDs correspond to multiple objects controlled by the same player, e.g. different units in CodeCraft or MicroRTS.